src/trio/_core/_tests/test_run.py

```diff
@@ -2554,6 +2563,26 @@ async def test_nursery_strict_exception_groups() -> None:
     assert exc.value.exceptions[0].args == ("foo",)
 
 
+async def test_nursery_loose_exception_groups() -> None:
+    """Test that loose exception groups can be enabled on a per-nursery basis."""
```

Comment:
This function tests that we get an `ExceptionGroup` when raising two errors in non-strict mode; can we add another which tests that raising a single error _doesn't_ result in a group?